src/llvm/tools/clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -12818,3 +12820,14 @@ Value *CodeGenFunction::EmitHexagonBuiltinExpr(unsigned BuiltinID,
 
   return nullptr;
 }
+
+Value *CodeGenFunction::EmitMSP430BuiltinExpr(unsigned BuiltinID,
+                                              const CallExpr *E) {
+  switch (BuiltinID) {
+  case Builtin::BI__builtin_flt_rounds:
+    // Rounding is to the nearest number.
+    return llvm::ConstantInt::get(IntTy, 1);
```

Comment:
Is this correct? Shouldn't we just lower to intrinsic call and do something in the backend (e.g. emit a library function call, etc.). Why it is not handled by the generic builtin lowering code?

Reply:
Thanks for the suggestion. Done.